mock-certify-plugin/pom.xml

```diff
@@ -84,7 +84,7 @@
         <dependency>
             <groupId>io.mosip.esignet</groupId>
             <artifactId>esignet-integration-api</artifactId>
-            <version>1.4.0</version>
+            <version>1.4.1</version>
             <exclusions>
                 <exclusion>
```

Comment:
All the version at the top also needs to be changed to 0.2.1-SNAPSHOT

Reply:
Changed the version to 0.2.1-SNAPSHOT for both mock-certify-plugin and mosip-identity-certify-plugin